shared/actions/login/index.js

```diff
@@ -420,25 +412,13 @@ export function openAccountResetPage () : AsyncAction {
 
 type SimpleCB = () => void
 function makeKex2IncomingMap (dispatch, getState, onBack: SimpleCB, onProvisionerSuccess: SimpleCB) : incomingCallMapType {
```

Comment:
This function can now be simplified away, but intentionally keeping the structure for now to simplify @MarcoPolo's sagaization work.